src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -210,4 +210,10 @@
   <data name="SARIF1019_RuleIdMustBePresentAndConsistent" xml:space="preserve">
     <value>In every result, at least one of the properties result.ruleId and result.rule.id must be present. If both are present, they must be equal.</value>
   </data>
+  <data name="SARIF1020_SchemaMustBePresentAndConsistent" xml:space="preserve">
+    <value>The schema must be presented and consistent.</value>
```

Comment:
>The schema must be presented and consistent. [](start = 11, length = 44)

"The $schema property should be present, and must refer to the final version of the SARIF 2.1.0 schema."